scripts/publish/aws-s3.sh

```diff
@@ -30,21 +30,24 @@ function usage() {
   echo "    -f <file>"
   echo "    -b <s3 bucket>"
   echo "    -v <version>"
-  echo "    -p <product name>"
+  echo "    -n <product name>"
+  echo "    -p S3 key prefix"
```

Comment:
Out of curiosity, why the backwards-incompatible switching of the `-p` flag to a different option? :-/